searchattribute/search_attribute.go

```diff
@@ -16,6 +16,13 @@ func NewSearchAttributeCommands() []*cli.Command {
 			Usage: common.CreateSearchAttributeDefinition,
 			UsageText: common.SearchAttributeCreateUsageText,
 			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     common.FlagNamespace,
+					Aliases:  common.FlagNamespaceAlias,
+					Required: true,
+					Usage:    "Temporal namespace",
+					Category: common.CategoryMain,
+				},
```

Comment:
I added the flag here because I could get from the global flags. Unsure which way is better. cc: @feedmeapples

Reply:
there are no more global flags, they have been moved under almost each individual command including the `search-attribute` ones

```
temporal operator search-attribute create -h                                                                                                                                                                                                                                                                                                                        

OPTIONS:
   Client Options:
....
   --namespace value, -n value              Temporal workflow namespace (default: "default") ....

   Display Options:
...
```

Reply:
Oh, so I don't need to add this flag here. Gonna remove it.